### PR TITLE
Json parsing for an unpresent param results in the string null, which…

### PR DESCRIPTION
**Pull request by gargshobhit90**
… is not -z

### CODE REVIEW

File: bin/out [hunk 1]
```diff
@@ -14,7 +14,7 @@ rollback() {
     DRY_RUN=$2
     [ -n "$DEPLOYMENT" ] || exit 1
     echo "Running: $KUBECTL rollout undo deployment/$DEPLOYMENT"
-    if [ -z "$DRY_RUN" ] || [[ "$DRY_RUN" = "false" ]]; then
+    if [[ "$DRY_RUN" == "null" ]] || [[ "$DRY_RUN" = "false" ]]; then
```
Comment (ilge) [inline bin/out]: does = "false" work the same way as == does?

Comment (gargshobhit90) [reply bin/out]: yeah in bash == is an alias of =
but, will make them the same (`==`)

Comment (gargshobhit90) [reply bin/out]: actually i think not the same for integers but same for strings

Comment (ilge) [reply bin/out]: tested and it seems to work. can consider using `==` in both for consistency.

**Pull request merged**